crates/fullnode/tests/runner_reorg_tests.rs

```diff
@@ -1,217 +0,0 @@
-use citrea_fullnode::CitreaFullnode;
-use sov_mock_da::{
-    MockAddress, MockBlob, MockBlock, MockBlockHeader, MockDaConfig, MockDaService, MockDaSpec,
-    MockValidityCond, PlannedFork,
-};
-use sov_mock_zkvm::MockZkvm;
-use sov_modules_api::default_context::DefaultContext;
-use sov_stf_runner::{
-    FullNodeConfig, InitVariant, RollupPublicKeys, RpcConfig, RunnerConfig, StorageConfig,
-};
-
-mod hash_stf;
-
-use hash_stf::{get_result_from_blocks, HashStf, Q, S};
-use sov_db::ledger_db::LedgerDB;
-use sov_mock_zkvm::MockCodeCommitment;
-use sov_prover_storage_manager::ProverStorageManager;
-use sov_rollup_interface::services::da::DaService;
-use sov_rollup_interface::storage::HierarchicalStorageManager;
-use sov_state::storage::NativeStorage;
-use sov_state::{ProverStorage, Storage};
-
-type MockInitVariant =
-    InitVariant<HashStf<MockValidityCond>, MockZkvm<MockValidityCond>, MockDaSpec>;
-#[tokio::test]
-#[ignore]
-async fn test_simple_reorg_case() {
```

Comment:
Maybe we should keep them?

Reply:
Yeah maybe, @eyusufatik  What do you think?